src/server/src/main/resources/db/cassandra/020_repair_run_tables.cql

```diff
@@ -0,0 +1,18 @@
+--
+--  Copyright 2019-2019 The Last Pickle Ltd
+--
+--  Licensed under the Apache License, Version 2.0 (the "License");
+--  you may not use this file except in compliance with the License.
+--  You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+--  Unless required by applicable law or agreed to in writing, software
+--  distributed under the License is distributed on an "AS IS" BASIS,
+--  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--  See the License for the specific language governing permissions and
+--  limitations under the License.
+--
+-- Upgrade to allow segments to contain several token ranges
+
+ALTER TABLE repair_run ADD tables text STATIC;
```

Comment:
`repair_unit_v1` uses `set<text>` types for table list storage. Any specific reason made you choose a plain text field instead?

Reply:
good point! i will change the code to use the `set<text>` approach.